install.sh

```diff
@@ -0,0 +1,98 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Not SUSE operating system."
+        return -1
+    fi
+    case "$VERSION_ID" in
+    "15.3")
+        repo="https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/devel:sap:trento.repo"
+        ;;
+    *)
+        echo "Not supported OS release."
+        return -1;;
+    esac
+    path=${repo%/*}/
+    if zypper lr --details | cut -d'|' -f9 | grep $path; then
+        echo "$path repository alreadt exists. Skipping."
+    else
+        zypper ar $repo
+    fi
+    zypper ref
+    if which trento; then
+        echo "trento is already installed. Skipping."
+    else
+        zypper in trento
+    fi
+}
+
+function install_consul() {
+    if which consul; then
+        echo "consul is already installed. Skipping."
+        return 0
+    fi
+    # Unzip cannot be extracted of the fly, so we store the file, extract and remove.
+    zipfile=$(mktemp)
+    # FIXME! The version is hardcoded. It should take the last existing.
+    wget https://releases.hashicorp.com/consul/1.9.8/consul_1.9.8_linux_amd64.zip -O $zipfile
```

Comment:
We could use also environment variables with a default value that if not overriden, gets the architecture using `arch` or `uname -m`

Reply:
This should of course be implemented, but I would prefer to do it already in another pull request because I'm afraid it will bring as much logic as the rest of the script. It would need to get somehow the latest version from the web page and interact with user, for example if the consul for his architecture was not found if he would like to install defaults.

Reply:
Yes, this is definitely for a follow up.